share/new_eds/testing.go

```diff
@@ -35,28 +36,31 @@ func TestSuiteAccessor(
 		t.Errorf("minSize must be power of 2: %v", maxSize)
 	}
 	for size := minSize; size <= maxSize; size *= 2 {
+		padding := rand.IntN(size)
```

Comment:
This random will select amount of padding in range of  [0;rowSize) So there always will be less than full row of padding. Lets ste up tests for cases to test instead:
- no padding
- at least 1 row.  Could be achieved by random in range [rowSize; 3/4 * rowSize*rowSize].